Point to docs for installation instructions (#727)
getsentry/sentry-dotnet: README.md

## Documentation

Each NuGet package in the table above has its custom view of the docs. Click on the badge to find the best documentation for your use case.

Sentry has extensive documentation for its SDKs on [https://docs.sentry.io](https://docs.sentry.io/platforms/dotnet/).
[The .NET API (DocFX) is generated on each merge to main and pushed to GitHub pages](https://getsentry.github.io/sentry-dotnet/index.html) Documentation.

### Samples

**Consider taking a look at the _[samples](https://github.com/getsentry/sentry-dotnet/tree/main/samples)_ directory for different types of apps and example usages of the SDK.**

> Looking for samples using the NuGet packages? Check out [this repository](https://github.com/getsentry/examples).

Author: ```suggestion
Looking for more samples? Check out [this repository](https://github.com/getsentry/examples).
```
